lms/djangoapps/courseware/toggles.py

```diff
@@ -38,3 +38,32 @@
 COURSEWARE_MICROFRONTEND_COURSE_TEAM_PREVIEW = CourseWaffleFlag(
     WAFFLE_FLAG_NAMESPACE, 'microfrontend_course_team_preview', __name__
 )
+
+COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES = ExperimentWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones', __name__)
+
+COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_FIRST_CELEBRATION = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones_first_celebration', __name__)
+
+COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_DISCUSSION = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones_discussion', __name__)
+
+COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_STREAKS = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones_streaks', __name__)
+
+
+def courseware_mfe_discussion_is_active(course_key):
```

Comment:
This is a much more generic name than I'd expect. Sounds like it's asking if the discussions are enabled at all in the MFE, not the celebration bit.